src/types/index.d.ts

```diff
@@ -338,3 +338,8 @@ export interface ItemTracked {
   itemId?: string;
   variationId?: string;
 }
+
+export interface ItemTrackedPurchased extends ItemTracked{
```

Comment:
```suggestion
export interface ItemTrackedPurchased extends ItemTracked {
```
Can we add a space before `{`?

Reply:
Yes, sure! Not exactly sure why Prettier didn´t pick on that though, will need to check.